elasticsearch/runserver

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+cd /usr/share/elasticsearch
+export ES_HEAP_SIZE=256M
```

Comment:
This should be configurable (at some point). If you don't feel like doing it in this PR, make an issue.

Reply:
Let me see how they do it for the official images.